packages/trpc/server/routers/viewer/organizations/update.handler.ts

```diff
@@ -24,10 +23,7 @@ export const updateHandler = async ({ ctx, input }: UpdateOptions) => {
   // A user can only have one org so we pass in their currentOrgId here
   const currentOrgId = ctx.user?.organization?.id || input.orgId;
 
-  if (!currentOrgId || ctx.user.role !== UserPermissionRole.ADMIN)
-    throw new TRPCError({ code: "UNAUTHORIZED" });
-
-  if (!(await isOrganisationAdmin(ctx.user?.id, currentOrgId)) || ctx.user.role !== UserPermissionRole.ADMIN)
+  if (!currentOrgId || !(await isOrganisationAdmin(ctx.user?.id, currentOrgId)))
```

Comment:
organization admin should be able to update organization.  ctx.user.role is different from organization's role

Reply:
I think that was there to let the instance admin be able to do whatever in an organization too

Reply:
okay. do we have to check if the instance admin is atleast member of organization ? Like

`
const isUserRoleAdmin = ctx.user.role === UserPermissionRole.ADMIN && (await isOrganisationMember(ctx.user?.id, currentOrgId))
`